composer.json

```diff
@@ -46,5 +56,14 @@
   },
   "scripts": {
     "cs": "vendor/bin/php-cs-fixer fix"
-  }
-}
\ No newline at end of file
+  },
+  "config": {
+    "sort-packages": true
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/brainexe/php-graph-sdk.git"
+    }
+  ]
```

Comment:
Shouldn't we omit this as well as the `dev-php8 as 5.7-dev` and let the user add it in their application `composer.json` like you stated in the readme?

Reply:
I think this entry is for the dev dependency.

Reply:
Makes sense. :+1: